packages/sdk/src/controllers/DataConnectorController.ts

```diff
@@ -0,0 +1,145 @@
+import { ConnectorConfigOptions, EditorAPI, EditorResponse, MetaData } from '../types/CommonTypes';
+import { getEditorResponseData } from '../utils/EditorResponseData';
+import { DataConnectorCapabilities, DataItem, DataPage, PageConfig } from '../types/DataConnectorTypes';
+
+/**
+ * The DataConnectorController is responsible for all communication regarding Data connectors.
+ * Methods inside this controller can be called by `window.SDK.DataConnector.{method-name}`
+ *
+ * The way CHILI Studio handles different sources of Data is called 'DataConnectors'. A DataConnectors is an
```

Comment:
Shouldn't it be GraFx Studio? @brapoprod ?

```suggestion
 * The way GraFx Studio handles different sources of Data is called 'DataConnectors'. A DataConnectors is an
```